### ISSUE TITLE
Fanv2 showing up as unsupported in the web interface

### ISSUE THREAD

**Issue by xiurobert** [oznu/homebridge-config-ui-x#418]
**Describe Your Problem:**

Fanv2 showing up as not supported on the web interface. Fan is configured from the homebridge-broadlink-rm plugin.

**Logs:**

https://hastebin.com/mijofirage.coffeescript
**Homebridge Config:**
https://hastebin.com/jusefeqepu.json

**Screenshots:**

https://i.imgur.com/Q3hm6Fa.png

**Environment:**

* **Node.js Version**:  v11.9.0
* **NPM Version**: 6.5.0
* **Homebridge Version**: 0.4.50
* **Homebridge Config UI X Version**: 4.6.3
* **Operating System**: Ubuntu Server 18.10 Cosmic Cuttlefish
* **Process Supervisor**: screen (basically nothing)

**Comment by oznu** [comment]
FanV2 is not currently a supported accessory type.

**Comment by xiurobert** [comment]
Sorry, let me rephrase for clarity. My accessory is a regular Fan, yet it is showing up as not supported in the web interface. This is rather strange. Is it because it is a Fanv2? I assumed that all fans were the same type

**Comment by oznu** [comment]
Fanv2 is a different accessory type to Fan in the HomeKit ecosystem so it needs to be implemented independently. I've linked this issue to the meta issue and I'll see how what's involved in adding it soon.
